packages/orm/src/loadLens.ts

```diff
@@ -75,7 +75,7 @@ function maybeLoad(object: any, path: string): unknown {
  *
  * This assumes you've first evaluated the lens with `loadLens` and now can access it synchronously.
  */
-export function getLens<T, U, V>(start: T, fn: (lens: Lens<T>) => Lens<U, V>): V {
+export function getLens<T, U, V>(start: T, fn: (lens: Lens<T>) => Lens<U, V>): V | V[] {
```

Comment:
Hm...I think that if the lens "resolved" to a collection, then `V` would already be an array...can this be removed?

Reply:
Leftover from an earlier iteration, will remove.